modules/applying-custom-seccomp-profile.adoc

```diff
@@ -0,0 +1,22 @@
+[id="applying-custom-seccomp-profile_{context}"]
+= Applying the custom seccomp profile to the workload
+
+.Prerequisite
+* The cluster administrator has set up the custom seccomp profile. For more details, see "Setting up the custom seccomp profile".
+
+.Procedure
+* Apply the seccomp profile to the workload by setting the `securityContext.seccompProfile.type` field as following:
```

Comment:
Should it be `seccompProfile` here?

Reply:
Here it just refers to the seccomp as a profile, not the field in the code.